src/.nuget/Microsoft.NETCore.ILDAsm/Microsoft.NETCore.ILDAsm.pkgproj

```diff
@@ -5,7 +5,7 @@
     <Version>1.0.3</Version>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
-    <PackagePlatforms>x64;x86;arm</PackagePlatforms>
+    <PackagePlatforms>x64;x86;arm64;arm</PackagePlatforms>
```

Comment:
orthogonal but something I notices while reviewing. This claims to support arm but I don't see an arm configuration below.